src/Components/WebAssembly/DevServer/src/Server/Program.cs

```diff
@@ -26,8 +26,8 @@ public class Program
             Host.CreateDefaultBuilder(args)
                 .ConfigureHostConfiguration(config =>
                 {
-                    var applicationPath = args.SkipWhile(a => a != "--applicationpath").Skip(1).FirstOrDefault();
-                    var applicationDirectory = Path.GetDirectoryName(applicationPath);
+                    var applicationPath = args.SkipWhile(a => a != "--applicationpath").Skip(1).First();
```

Comment:
I guess this is fine but I feel like `FirstOrDefault` is more defensive in case a user is invoking the tool without the correct arguments. Not likely but doesn't hurt. 🤷🏽‍♀️

Reply:
The parameter is required, so we have to fail it at this point (you get all sorts of arg null errors if this is default). We could return a non-zero exit code, but the exception does the same